deployment.template.json

```diff
@@ -48,7 +48,7 @@
             "restartPolicy": "always",
             "settings": {
               "image": "${MODULES.CameraCapture.arm32v7}",
-              "createOptions": "{\"Env\":[\"VIDEO_PATH=0\",\"IMAGE_PROCESSING_ENDPOINT=http://image-classifier-service:80/image\", \"VERBOSE=TRUE\", \"RESIZE_WIDTH=256\", \"RESIZE_HEIGHT=256\"], \"HostConfig\":{\"Binds\":[\"/dev/video0:/dev/video0\"],\"Privileged\":true}}"
+              "createOptions": "{\"Env\":[\"VIDEO_PATH=0\",\"IMAGE_PROCESSING_ENDPOINT=http://image-classifier-service:80/image\", \"VERBOSE=TRUE\", \"RESIZE_WIDTH=256\", \"RESIZE_HEIGHT=256\", \"SHOW_VIDEO=TRUE\"], \"HostConfig\":{\"PortBindings\":{\"5012/tcp\":[{\"HostPort\":\"5012\"}]},\"Binds\":[\"/dev/video0:/dev/video0\"],\"Privileged\":true}}"
```

Comment:
I assume that the camera-capture module will still run fine if showVideo=true but that no browser is connected to the webserver. Can you confirm? Thanks